redesign.user.js

```diff
@@ -797,14 +797,69 @@ const createSidebar = (id, position, icon, callback) => {
         document.getElementById('page')?.before(sidebar);
 
         // append the toggle button
-        document
-            .querySelector('#page .drawer-toggles')
-            ?.append(toggleBtnWrapper);
+        const togglesDiv =
+            document.querySelector(
+                `.${PREFIX(`drawer-toggles-${position}`)}`
+            ) ||
+            (() => {
+                const togglesDiv = document.createElement('div');
+                togglesDiv.classList.add(PREFIX(`drawer-toggles-${position}`));
+                document
+                    .querySelectorAll(`.drawer-${position}-toggle`)
+                    .forEach(toggler => {
+                        togglesDiv.append(toggler);
+                    });
+                document
+                    .querySelector('#page .drawer-toggles')
+                    .append(togglesDiv);
+                return togglesDiv;
+            })();
```

Comment:
Not necessarily required as it would only be relevant when there are hundreds or thousands or hundreds of thousands sidebars were added.
Storing the created toggles in a Map would reduce runtime as heavy DOM-Operations are reduced.

Reply:
I am now using `getElementByID` (see https://github.com/YorikHansen/better-moodle/commit/9d9f660866c6faadc374d461c6efccb841cd2ad2). This is better than using another Map object. Therefore this is resolved